project/AppDependencies.scala

```diff
@@ -3,30 +3,30 @@ import play.core.PlayVersion
 
 object AppDependencies {
 
-  val bootstrapVersion = "7.12.0"
+  val bootstrapVersion = "7.19.0"
 
   private lazy val compile = Seq(
     play.sbt.PlayImport.ws,
-    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-play-28"             % "0.74.0",
-    "uk.gov.hmrc"             %% "play-frontend-hmrc"             % "6.2.0-play-28",
-    "uk.gov.hmrc"             %% "domain"                         % "8.1.0-play-28",
-    "uk.gov.hmrc"             %% "play-conditional-form-mapping"  % "1.12.0-play-28",
+    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-play-28"             % "1.3.0",
```

Comment:
This can be reverted back to the previous version since we have a ticket for the mongo upgrades.